### PR TITLE
Add new cytonic chains

### PR DESCRIPTION
**Pull request by GooGrand**
Added new supported chains for the cytonic protocol.

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/cytonic: 
        
 
  
    
    
    Error: 401 Unauthorized: {"jsonrpc":"2.0","error":{"code":-32401,"message":"Unverified free plans are temporarily disabled due to abuse. Please verify your email to regain access to project beginning with 3c98f..."}}
    Truncated error stack:
        at ClientBrowser.callServer (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@solana/web3.js/lib/index.cjs.js:4944:18)

**Comment by llamatester** [comment]
The adapter at projects/cytonic exports TVL: 
        
 
 
    ethereum                  436.13 k
    base                      206.50 k
    solana                    200.29 k
    arbitrum                  134.24 k
    avax                      89.29 k
    optimism                  74.20 k
    bsc                       41.27 k
    blast                     18.90 k
    polygon                   9.17 k
    manta                     30
    bsquared                  2
    
    total                    1.21 M

**Comment by llamatester** [comment]
The adapter at projects/cytonic exports TVL: 
        
 
 
    ethereum                  436.07 k
    base                      206.52 k
    solana                    200.29 k
    arbitrum                  134.23 k
    avax                      89.31 k
    optimism                  74.19 k
    bsc                       41.27 k
    blast                     18.90 k
    polygon                   9.18 k
    manta                     30
    bsquared                  2
    
    total                    1.21 M

**Comment by llamatester** [comment]
Error while running adapter at projects/stargaze: 
        
 
  
    
    
    Error: Api response include NFT value
    Truncated error stack:
    
        at tvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/stargaze/index.js:5:9)
        at getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:52:29)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:173:15
        at Array.map (<anonymous>)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:161:29
        at Array.map (<anonymous>)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:156:44
        at Object.<anonymous> (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:259:3)

**Comment by llamatester** [comment]
The adapter at projects/cytonic exports TVL: 
        
 
 
    ethereum                  436.50 k
    base                      206.74 k
    solana                    200.49 k
    arbitrum                  134.31 k
    avax                      89.93 k
    optimism                  74.26 k
    bsc                       41.29 k
    blast                     18.92 k
    polygon                   9.18 k
    manta                     30
    bsquared                  2
    
    total                    1.21 M

File: projects/helper/env.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const DEFAULTS = {
   EVMOS_MULTICALL_CHUNK_SIZE: "3", // evmos reduced gas limit, this is a workaround to make multicall work
   STARKNET_RPC: 'https://starknet-mainnet.public.blastapi.io',
   COVALENT_KEY: 'ckey_72cd3b74b4a048c9bc671f7c5a6',
-  SOLANA_RPC: 'https://mainnet.helius-rpc.com/?api-key=0109717a-77b4-498a-bc3c-a0b31aa1b3bf',
+  SOLANA_RPC: 'https://mainnet.helius-rpc.com/?api-key=7873b843-941b-45f1-a402-6a41f370ea30',
```
Comment (g1nt0ki) [inline projects/helper/env.js:10]: can you revert this? we are fine/use our own rpc but this leaks your api key to public

**Comment by llamatester** [comment]
Error while running adapter at projects/cytonic: 
        
 
  
    
    
    Error: 401 Unauthorized: {"jsonrpc":"2.0","error":{"code":-32401,"message":"Unverified free plans are temporarily disabled due to abuse. Please verify your email to regain access to project beginning with 3c98f..."}}
    Truncated error stack:
        at ClientBrowser.callServer (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@solana/web3.js/lib/index.cjs.js:4944:18)

File: projects/helper/env.js (hunk 1 above)
Comment (GooGrand) [reply projects/helper/env.js]: @g1nt0ki and it doesn't work rn

**Pull request merged**